docs/src/test/java/docs/home/persistence/CqrsIntegrationTest.java

```diff
@@ -114,6 +114,7 @@ private void eventually(Effect block) {
   }
 
   @Test
+  @Ignore
```

Comment:
Did you mean to leave the `@Ignore` here? That could certainly explain it not failing! :wink:

Reply:
Nah, it must be something else. :-P